processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -80,10 +80,6 @@ public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
   @Override
   public ValueMatcher makeMatcher(ColumnSelectorFactory factory)
   {
-    if (filters.size() == 0) {
-      return BooleanValueMatcher.of(false);
```

Comment:
`makeMatcher(ValueMatcher[])` has `Preconditions.checkState(baseMatchers.length > 0);`, so it should end up throwing an exception anyway